plugin.js

```diff
@@ -5,27 +5,80 @@ var fs = require("fs");
 var path = require("path");
 var AdmZip = require("adm-zip");
 var events = require("events");
+var rmdir = require("rimraf"); // I still don't believe this is really a thing
+var pluginParser = require('./plugin-parser.js');
 
 var fileName = null;
+var localState = null;
```

Comment:
localState here is used to give us the ability to update the Ractive instance that contains the list of plugins. In this function, it is used to change the "selected" variable that Ractive uses to create the contents of the plugin popup.